documentation/concepts/device_scheduler.rst

```diff
@@ -0,0 +1,151 @@
+:orphan:
+
+.. _device_scheduler:
+
+Device Scheduler
+===========
+
+Introduction
+--------------
+This generic device scheduler is able to handle an EMS with multiple devices, with various types of constraints on the EMS level and on the device level,
+and with multiple market commitments on the EMS level.
+
+A typical example is a house with many devices. The commitments are assumed to be with regard to the flow of energy to the device (positive for consumption, negative for production). In practice, this generic scheduler is used in the **StorageScheduler** to schedule a storage device.
+    
+The solver minimises the costs of deviating from the commitments.
+
+
+
+Notation
+---------
+
+================================ ================================================ ==============================================================================================================  
+Symbol                              Variable in the Code                           Description
```

Comment:
This list could be split up into variables and parameters (pyomo terminology, I think). Variables are decision variables whose value needs to be found and parameters are known input values.